rules/windows/privilege_escalation_printspooler_suspicious_spl_file.toml

```diff
@@ -0,0 +1,44 @@
+[metadata]
+creation_date = "2020/08/14"
+ecs_version = ["1.5.0"]
+maturity = "production"
+updated_date = "2020/08/14"
+
+[rule]
+author = ["Elastic"]
+description = """
+Detects exploitation attempt behavior including on a patched system of 2 Privilege Escalation vulnerabilities related to
+the Print Spooler service. For more information lookup the related VulnIDs CVE-2020-1048, CVE-2020-1337 and verify that
+the impacted system is patched.
```

Comment:
I would make the verbiage similar to #146 and expand to describe the file writes to differentiate the two

Reply:
done